src/app/CASESessionManager.h

```diff
@@ -124,15 +125,16 @@ class CASESessionManager : public OperationalSessionReleaseDelegate, public Sess
      * If the connection is already established, the `onConnection` callback will be immediately called,
      * before `FindOrEstablishSession` returns.
      *
-     * The `attemptCount` parameter can be used to automatically retry multiple times if session setup is
-     * not successful.
+     * The `attemptCount` parameter can be set to a value greater than 1 to automatically make
+     * at least attemptCount session establishment attempts until session setup is successful.
```

Comment:
This part is definitely backing out previous changes.  Please don't do that.

Reply:
Yes, something went wrong with merging master to my branch.... I will revert that.